.Documentation/articles/getting-started/designer.md

```diff
@@ -105,13 +105,13 @@ Now, click on the first step to open it in the [Step Inspector](../innoactive-cr
 
 #### Step Inspector
 
-The [Step Inspector](../innoactive-creator/step-inspector.md) allows you to set a step name and description, as well as to create and edit behaviors, transitions, and conditions. The name and description of the step have no effect on the training course itself. Use them to keep notes for yourself and other trainers. To add a behavior, click on an `Add Behavior` button in the step's view. To delete a behavior, click on the `[x]` button next to it. The same applies to transitions and conditions.
+The [Step Inspector](../innoactive-creator/step-inspector.md) allows you to set a step name and description, as well as to create and edit behaviors, transitions, and conditions. The name and description of the step have no effect on the training course itself. Use them to keep notes for yourself and other trainers. To add a behavior, click on an `Add Behavior` button in the step's view. To delete a behavior, click on it's button with the trash icon. The same applies to transitions and conditions.
```

Comment:
Liked the phrasing with "click on the trash button next to it" better, but doesn't matter too much

Reply:
I changed it because I did not like the next to it. I see the trash icon as part of it instead of it being next to it 🤔  but I can also leave it with the cenxt to it. What do you think?

Reply:
I found another description in the Step Inspector md, and I copied that one for consistency :)